packages/driver/cypress/e2e/commands/actions/type_events.cy.js

```diff
@@ -594,26 +601,32 @@ describe('src/cy/commands/actions/type - #type events', () => {
 
       targets.forEach((target) => {
         it(target, () => {
-          cy.get(`#target-${target}`).focus().type('{enter}')
-
-          cy.get('li').should('have.length', 4)
-          cy.get('li').eq(0).should('have.text', 'keydown')
-          cy.get('li').eq(1).should('have.text', 'keypress')
-          cy.get('li').eq(2).should('have.text', 'click')
-          cy.get('li').eq(3).should('have.text', 'keyup')
+          // TODO fix this test in Webkit https://github.com/cypress-io/cypress/issues/26438
+          if (!isWebkit) {
```

Comment:
@astone123 You can [use test configuration](https://github.com/cypress-io/cypress/blob/89c23e08de76ff37093ac6c6958a8ffdaecc4fc0/packages/driver/cypress/e2e/commands/navigation.cy.js#L766) to keep tests from running in webkit instead of a condition inside the test.

Reply:
nice 👍🏻